### DISCUSSION TITLE
Keep Screen Awake Function

### DISCUSSION THREAD

**Discussion by robweber** [mealie-recipes/mealie#2628] [Q&A]
In the demo version of Mealie there is a "keep screen awake" toggle on each recipe. I've deployed the latest Docker image locally and my recipes don't have this toggle available. I've dug through the documentation and settings and just can't seem to find how to enable this. If anyone can point me in the right direction I'd appreciate it - my family was really looking forward to that function to keep phones/tablets active when cooking.

**Comment by ghuser238490** [comment]
It's a confirmed bug: https://github.com/mealie-recipes/mealie/issues/2305

**Comment by robweber** [comment]
That issue says that the toggle doesn't work, on my install I don't even have the toggle. Was it removed due to the bug? If so why is it on the demo? I'm not using iOS and the testing page provided in that bug report works for me.

**Reply by Symbiot78** [reply]
same here via browser on mobile and on computer browser.

**Reply by Symbiot78** [reply]
I see reference to safari on ios working. I checked via chrome.
on my install the button is not present.
